test/ex_doc/autolink_test.exs

```diff
@@ -1,5 +1,5 @@
 defmodule ExDoc.AutolinkTest do
-  use ExUnit.Case, async: true
+  use ExUnit.Case
```

Comment:
if we test all warnings in that one test we could run it concurrently again, should we do that?

Reply:
Tests do not run concurrently within the same module, regardless of the `:async` option, so that wouldn't be the limitation, if that is what you mean.
Concurrency was disabled because there are two ExUnit cases that are capturing :stderr, the other one is  `ExDoc.CLITest`

Reply:
It is supported on Elixir v1.11 though so let's leave the option in and push everyone to more recent Elixir versions.

Reply:
> It is supported on Elixir v1.11 though so let's leave the option in and push everyone to more recent Elixir versions.
@josevalim Sorry, but what is supported in v1.11?

Reply:
> if we test all warnings in that one test we could run it concurrently again, should we do that?

Do we really have a limitation having different tests emitting a warning since they are in the same test case?